src/main/kotlin/com/openlattice/shuttle/util/CsvUtil.kt

```diff
@@ -0,0 +1,18 @@
+package com.openlattice.shuttle.util
+
+import com.fasterxml.jackson.dataformat.csv.CsvMapper
+import com.fasterxml.jackson.dataformat.csv.CsvSchema
+
+class CsvUtil {
+    companion object {
+        @JvmStatic
+        fun newDefaultMapper(): CsvMapper {
+            return CsvMapper()
```

Comment:
(I know you just moved this but)
I thiiiiink this is fully threadsafe, so maybe we should make this into a singleton [ex: `ObjectMappers`]